src/lib.rs

```diff
@@ -78,6 +85,13 @@ enum Icon {
     Pencil,
 }
 
+/// HUD Actions
+///
+/// Events within the HUD that are returned back to the user.
+pub enum HUDAction {
+    
+}
```

Comment:
I think it will make more sense if `Hud` is a variant on an enum called `Action`, since the HUD will only produce a single type of event.